### PR TITLE
DEVX-710: build topic/producer/consumer map

### PR DESCRIPTION
**Pull request by ybyzek**

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -205,6 +205,70 @@ Topic Management
       .. figure:: images/create_topic.png
          :alt: image
 
+7. When Confluent Monitoring Interceptors are configured on any Kafka-based clients, i.e., producers, consumers, Connect, KSQL, then they write metadata to a topic called ``_confluent-monitoring``.
+   Confluent Control Center reads that topic to do expert stream monitoring for message delivery assurance and performance for throughput and latency.
```
Comment (joel-hamill) [inline docs/index.rst]: suggest using text-string replace for C3: `|c3|` or the abbreviated `|c3-short|`

File: docs/index.rst [hunk 2]
```diff
@@ -205,6 +205,70 @@ Topic Management
       .. figure:: images/create_topic.png
          :alt: image
 
+7. When Confluent Monitoring Interceptors are configured on any Kafka-based clients, i.e., producers, consumers, Connect, KSQL, then they write metadata to a topic called ``_confluent-monitoring``.
```
Comment (joel-hamill) [inline docs/index.rst]: suggest: 

```
When Confluent Monitoring Interceptors are configured on any Kafka-based clients, they write metadata to a topic called ``_confluent-monitoring``. Kafka-based clients can include producers, consumers, Connect, and KSQL.
```

File: docs/index.rst (hunk 1 above)
Comment (mcqueary) [reply docs/index.rst]: I would change `i.e.` to `e.g.` and drop the trailing comma.

Comment (ybyzek) [reply docs/index.rst]: Ack

File: docs/index.rst (hunk 2 above)
Comment (ybyzek) [reply docs/index.rst]: Ack

File: docs/index.rst [hunk 3]
```diff
@@ -205,6 +205,70 @@ Topic Management
       .. figure:: images/create_topic.png
          :alt: image
 
+7. When Confluent Monitoring Interceptors are configured on any Kafka-based clients, they write metadata to a topic called ``_confluent-monitoring``. Kafka-based clients include producers, consumers, Connect, KSQL, etc.
```
Comment (mcqueary) [inline docs/index.rst]: Is there a compelling reason that we say "Kafka-based clients" vs. just "Kafka clients"? For instance if I were referring to TIBCO EMS, I would simply say "EMS client" and likewise for NATS I would say "NATS client".

Comment (mcqueary) [reply docs/index.rst]: Consider removing the word "any" -- it's unnecessary here.

Comment (mcqueary) [reply docs/index.rst]: Consider changing the description of Kafka clients as follows, for clarity and precision: "Kafka clients include any application that uses the Apache Kafka client API to connect to the Kafka broker as producers or consumers, such as custom clients, Connect, KSQL, or |c3|"

File: docs/index.rst [hunk 4]
```diff
@@ -205,6 +205,70 @@ Topic Management
       .. figure:: images/create_topic.png
          :alt: image
 
+7. When Confluent Monitoring Interceptors are configured on any Kafka-based clients, they write metadata to a topic called ``_confluent-monitoring``. Kafka-based clients include producers, consumers, Connect, KSQL, etc. 
+   |c3| reads that topic to do expert stream monitoring for message delivery assurance and performance for throughput and latency.
```
Comment (mcqueary) [inline docs/index.rst]: Hard to parse. One too many "and"s or "for"s in here, or perhaps one too few verbs, to understand what is modifying what, grammar-wise. Lest you think I'm being snarky, I'm not -- I do not fully understand this sentence as written.

File: docs/index.rst [hunk 5]
```diff
@@ -205,6 +205,70 @@ Topic Management
...
+   From that topic, you can also verify which producers are writing to which topics and which consumers are reading from which topics.
+   Checkout the sample script provided into the repo (Note: this is for demo purposes only, not suitable for production).
```
Comment (mcqueary) [inline docs/index.rst]: Consider saying "An example script is provided with the repo [...]".

File: docs/index.rst (hunk 3 above)
Comment (ybyzek) [reply docs/index.rst]: @mcqueary : spot on -- many users do not realize that Kafka Connect (for example) has embedded producers and consumers.  

I have removed `Kafka-based` and reworded very closely to your suggestion.

Thanks!

File: docs/index.rst (hunk 5 above)
Comment (ybyzek) [reply docs/index.rst]: Ack

File: docs/index.rst (hunk 4 above)
Comment (ybyzek) [reply docs/index.rst]: Ack, thanks

**Comment by ybyzek** [comment]
@mcqueary PR good to go?
